docs/FAQ.md

```diff
@@ -149,27 +149,55 @@ and [give feedback](https://join.slack.com/t/tigerbeetle/shared_invite/zt-1gf3qn
-
-3. Isolation: All events within a batch, across all clients, are
-   serialized into TigerBeetle's state machine. There are no
-   concurrent events in TigerBeetle, even across clients.
-
-4. Durability: Absolute durability is impossible, because all hardware
-   can ultimately fail. Data we write today might not be available
-   tomorrow. TigerBeetle embraces limited disk reliability and
-   maximizes data durability in spite of imperfect disks. We actively
-   work against such entropy by taking advantage of cluster-wide
-   storage. A bit of data would need to get corrupted on all replicas
-   in a cluster to get lost, and even in that case the system would
-   safely halt.
+### Atomicity
+
+As part of replication, each operation is durably stored in a majority
+of replicas' Write-Ahead Log before indicating to the primary that the
+operation is successful. Only over time do WAL entries make their way
+through a state machine and into the LSM storage layer.
+
+The WAL is what allows TigerBeetle to achieve atomicity and durability
+since once the WAL is the source of truth. If TigerBeetle crashes, the
+WAL is replayed on startup against the last checkpoint on disk.
+
+Financial atomicity goes further than this though: events and
+transfers can be linked when created so they all succeed or fail
+together.
+
+### Consistency
+
+TigerBeetle guarantees strict serializability. And at the cluster
+level, stale reads are not possible since all operations (i.e. reads,
+not just writes) go through consensus.
+
+However, financial consistency requires more than this. TigerBeetle
+exposes a double-entry accounting API where money is never created or
+destroyed, only transferred between accounts. And transfer history is
+immutable. You can read more about our consistency guarantees
+[here](./design/consistency.md).
+
+### Isolation
+
+All events within a batch (across all clients) are
+serialized at the state machine level. There are no concurrent
+events in TigerBeetle, even across clients.
+
+### Durability
+
+Up until 2018, traditional DBMS durability has focused on the crash
+consistency model, however, Fsyncgate and [Protocol Aware
+Recovery](https://www.usenix.org/conference/fast18/presentation/alagappan)
+have shown that this model can lead to real data loss for users in the
+wild. TigerBeetle therefore adopts an explicit storage
+fault model, which we then verify and test with incredible levels of
+corruption, something which few distributed systems
+were ever designed to handle. Our emphasis on protecting Durability is
+what sets TigerBeetle apart, not only as a ledger but as a DBMS.
+
+However, absolute durability is impossible, because all hardware can
+ultimately fail. Data we write today might not be available
+tomorrow. TigerBeetle embraces limited disk reliability and maximizes
+data durability in spite of imperfect disks. We actively work against
+such entropy by taking advantage of cluster-wide storage. A bit of
```

Comment:
Is "bit" being used colloquially here, or as a literal 1/8-of-a-byte? If the former I'd use a different word.

Reply:
This was @matklad 's phrase. I also am not completely sure if it was meant as a literal bit or rather "a chunk".

Reply:
Colloquial sense, but I now see that this is ambiguous. Maybe just call it a "record"? 

> A record can only be lost if it gets corrupted on all replica, and even in that case the system would safely halt.

Reply:
Switched to record. Thanks!